pages/index.tsx

```diff
@@ -23,7 +22,29 @@ interface ClassData {
 	course_id: string
 }
 
-const Home: NextPage = () => {
+type ClassWithId = Class & { id: number }
+
+// use axios.get in the getStaticProps method from Next.js to get data from  https://omshub-readonly.gigalixirapp.com/classes
+export const getStaticProps: GetStaticProps = async () => {
+	const res = await fetch('https://omshub-api.gigalixirapp.com/api/classes')
```

Comment:
We should extract the link to an env var (e.g., `REACT_APP_BACKEND_URL`), which could be referenced here as `process.env.REACT_APP_BACKEND_URL + '/classes`, or similar (and similar in general for API calls). May also make sense wrap the network call in a `try`-`catch` block here for good measure..

Reply:
If the PR is improving the site, why kill it because it does not improve it even more?